.fantomasignore

```diff
@@ -0,0 +1,5 @@
+test/
+build/
+benchmarks/
+utils/
+demo.fsx
```

Comment:
Replicated from `Directory.Solution.targets`.  We should probably start considering formatting these at some point.